extensions/query-engine/src/test/java/com/aerospike/helper/query/SelectorTests.java

```diff
@@ -100,6 +97,44 @@ public void selectEndsWith() throws IOException {
 			it.close();
 		}
```

Comment:
current implementation of checks in other tests is tricky. 
in case iterator does not have any elements -- tests will still pass :)